app/(gcforms)/[locale]/(form administration)/form-builder/[id]/preview/Preview.tsx

```diff
@@ -42,15 +43,14 @@ export const Preview = ({
     getSecurityAttribute: s.getSecurityAttribute,
   }));
 
-  // TODO probably redirecting to the error page makes more sense since the error is not recoverable
-  const formParsed = safeJSONParse(getSchema());
-  if (formParsed?.error) {
+  const formParsed = safeJSONParse<FormProperties>(getSchema());
+  if (!formParsed) {
     toast.error(<ErrorSaving errorCode={FormServerErrorCodes.JSON_PARSE} />, "wide");
```

Comment:
How to handle this unlikely error case needs more thought.
The problem is the error message shown will mention a JSON error and give the option to download the file. But the file will be the default created form and be valid JSON. (hiding the issue of invalid JSON)

To resolve this the component code probably needs to be refactored to handle an error case like this better.